include/SFML/Window/Keyboard.hpp

```diff
@@ -225,6 +460,10 @@ class SFML_WINDOW_API Keyboard
 /// {
 ///     // quit...
 /// }
+/// else if (sf::Keyboard::isKeyPressed(sf::Keyboard::ScanGrave))
+/// {
+///     // open in-game command line
```

Comment:
`isKeyPressed()` will return `true` repeatedly as long as the key is held down, so I'm not sure if it's a good candidate for one-time-actions like in-game command line. For "quit" it's typically okay, as this skips the next polling of the keyboard state.

Maybe we can find a different action (e.g. "fire" or something continuous), or explicitly mention statefulness as in "open command-line, if not already open".

Reply:
The goal here was to demonstrate the use of a key that you couldn't really address before and I thought of the quake/Half-Life console opening. If you have different ideas with a "special/new" key, let me know

Reply:
I think it's OK, we should just avoid that people think it checks for a one-time key press.

As mentioned, maybe this is already enough, or is the hint too subtle?
```cpp
// open in-game command line (if it's not already open)
```